### DISCUSSION TITLE
Category settings

### DISCUSSION THREAD

**Discussion by lemte** [stoatchat/discussions#745] [Feature Suggestions]
When I refer to this topic, I mean the following questions:

1. Permissions

- The only roles allowed to view the category with the channels inside.
- The only roles capable of changing the permissions of this category.

2. Synchronizing permissions   

- The permissions configured in the category synchronize with all types of channel within that category.
- Faster/easier organization when creating a category and configuring it.

**Comment by vizthex123** [comment]
Kinda surprised it's not already a thing, but I would also love to have it. Much easier to manage 1 category's perms rather than editing all the channels if I want to make a minor change.

**Comment by DevSmoger** [comment]
Pump, Pump, Pump it UP.
I was also looking permissions for the category. You can hide channels inside, but not the categories. Can't make hidden private categories this way, everyone who joins knows there is something hidden. It would be way easier to add permissions in server. Right now it takes more time to set for each role and channel these permissions. 
Would say devs have done overall good work with Revolt chat.
